Source/JavaScriptCore/runtime/LiteralParser.cpp

```diff
@@ -703,10 +703,11 @@ static constexpr const bool safeStringLatin1CharactersInStrictJSON[256] = {
 template <typename CharType>
 static ALWAYS_INLINE bool isJSONWhiteSpace(const CharType& c)
 {
-    return isLatin1(c) && tokenTypesOfLatin1Characters[c] == TokErrorSpace;
+    return tokenTypesOfLatin1Characters[static_cast<uint8_t>(c)] == TokErrorSpace && isLatin1(c);
```

Comment:
Why make this change?

Reply:
Because this is almost always ASCII characters (otherwise, it becomes SyntaxError). So this is better ordering.